src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java

```diff
@@ -366,7 +367,12 @@ private static StringBuilder getCaseCauseListByRoom(ListingData listingData, Str
     public static StringBuilder getListingDate(ListingData listingData) {
         var sb = new StringBuilder();
         if (listingData.getHearingDateType() != null
-                && listingData.getHearingDateType().equals(RANGE_HEARING_DATE_TYPE)) {
+                && RANGE_HEARING_DATE_TYPE.equals(listingData.getHearingDateType())
+                && SERVING_CLAIMS_REPORT.equals(listingData.getReportType())) {
+            return getServedClaimsReportPeriod(listingData);
+
+        } else if (listingData.getHearingDateType() != null
+                && RANGE_HEARING_DATE_TYPE.equals(listingData.getHearingDateType())) {
```

Comment:
Part of the advantage of using String.equals is that you don't need the preceding null check